assets/css/src/amp-validation-error-taxonomy.css

```diff
@@ -362,3 +364,137 @@ body.taxonomy-amp_validation_error .wp-list-table .new th.check-column input {
 	font-weight: bold;
 	text-align: right;
 }
+
+/* Stylesheets */
+.amp-stylesheet-summary {
+	margin-bottom: 1em;
+}
+
+.amp-stylesheet-summary th {
+	text-align: right;
+	padding-right: 1ex;
+	font-weight: normal;
+}
+
+.amp-stylesheet-summary td {
+	font-weight: bold;
+	text-align: right;
+}
+
```

Comment:
Maybe adding something like this would help:

```css
.amp-stylesheet-list th {
    width: 15%;
}
```

Without it, the columns are pretty narrow on a 16" display:

<img width="1338" alt="columns-narrow-here" src="https://user-images.githubusercontent.com/4063887/73054157-4b024580-3e4f-11ea-8e00-a8577b99d3b9.png">

Reply:
The stylesheet actually already has:

![image](https://user-images.githubusercontent.com/134745/73081236-18bb0d00-3e7c-11ea-8b09-bde200107bb4.png)

Please try clearing/disabling cache and check again.

Reply:
Ah, sorry. The styling looks fine with a hard refresh:

<img width="910" alt="refresh-hard" src="https://user-images.githubusercontent.com/4063887/73091264-b4aa4000-3e9f-11ea-8d32-315387dd7824.png">